ibm_db2/assets/configuration/spec.yaml

```diff
@@ -56,6 +56,16 @@ files:
           has to be accessible and readable by the agent user.
         value:
           type: string
+      - name: extra_connection_parametters
```

Comment:
do you have other examples of parameters that might be passed here? Otherwise I'd rather have a config option just for timeout

Reply:
you have all those parametters https://www.ibm.com/docs/en/db2/11.1?topic=file-data-server-driver-configuration-keywords

Reply:
I think it may be safer to just include the timeout parameter, since adding this catch-all config for all other possible parameters could lead to unknown behavior that we don't know yet.

Reply:
In my last commit, I made the changes to only accept a timeout.